lib/amountItems/FixedIncomeProducts.js

```diff
@@ -6,47 +6,55 @@ import {numberPad, fillWith, fillWithPattern} from '../utils.js';
  * FixedIncomeProducts Class
  *
  * Produits de placement à revenu fixe
- * R237 - R239
+ * R237 - R238
+ *
+ * Produits des minibons et des prêts, financement participatif
```

Comment:
Actually crowdfunding is a different product from fixed income, I think we should handle it in a separate class.